modules/probe.sh

```diff
@@ -38,6 +38,8 @@ probe_arch() {
     target_arch="unknown"
     if grep -q x86-64 <<< "${file_info}"; then
         target_arch="x86_64"
+    elif grep -q "Intel 80386" <<< "${file_info}"; then
```

Comment:
I chose this string based on my experiments:

```
$file -b /bin/nano
ELF 32-bit LSB pie executable, Intel 80386, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux.so.2, BuildID[sha1]=a007931407fcfdcd7d640f588e53fe50a944a095, for GNU/Linux 3.2.0, stripped
```

And that is equivalent to the `x86_64` above.

But we could add something more strict like `ELF 32-bit.*386` (and for consistency `ELF 64-bit.*x86_64` above).

Reply:
I'd vote for the strict version, testing `ELF 32-bit.*` and `ELF 64-bit.*`. However, I don't consider it a blocker.